Sources/Model/Settings.swift

```diff
@@ -65,15 +65,18 @@ import Photos
         ]
     }
 
-    public class Selection : NSObject {
+   @objc public class Selection : NSObject {
         /// Max number of selections allowed
-        public lazy var max: Int = Int.max
+       @objc public lazy var max: Int = Int.max
         
         /// Min number of selections you have to make
-        public lazy var min: Int = 1
+       @objc public lazy var min: Int = 1
         
         /// If it reaches the max limit, unselect the first selection, and allow the new selection
-        public lazy var unselectOnReachingMax : Bool = false
+       @objc public lazy var unselectOnReachingMax : Bool = false
+    
+       /// If it reaches the max limit, notify
+       @objc public lazy var notifyReachingMaxLimit : Bool = false
```

Comment:
To follow the naming of the previous property...maybe change to notifyOnReachingMax

Reply:
resolved